### PR TITLE
docs(v2): update installation docs

### PR DESCRIPTION
**Pull request by iamayushdas**
# Motivation 

I was upgrading the docusaurus and saw that new release has been done, and i checked the installation is not updated

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

**Comment by Josh-Cena** [comment]
In fact the website is not incorrect:

> Please update to the latest Docusaurus 2 version shown at the top of the page, not what is shown below.

The version here is only an example.

**Comment by iamayushdas** [comment]
> In fact the website is not incorrect:
> 
> > Please update to the latest Docusaurus 2 version shown at the top of the page, not what is shown below.
> 
> The version here is only an example.

yes, I triggered a problem while updating from 2.0.0beta.0 to 2.0.0beta.2 , documentations not loading at all

File: website/docs/installation.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ You should see the correct version as output.
 Alternatively, if you are using Yarn, you can do:
 
 ```bash
-yarn upgrade @docusaurus/core@2.0.0-beta.0 @docusaurus/preset-classic@2.0.0-beta.0
+yarn upgrade @docusaurus/core@2.0.0-beta.2 @docusaurus/preset-classic@2.0.0-beta.2
```
Comment (slorber) [inline website/docs/installation.md:154]:
```suggestion
yarn upgrade @docusaurus/core@latest @docusaurus/preset-classic@latest
```

File: website/docs/installation.md [hunk 2]
```diff
@@ -128,8 +128,8 @@ Please update to the latest Docusaurus 2 version shown at the top of the page, n
 
 ```json title="package.json"
 "dependencies": {
-  "@docusaurus/core": "^2.0.0-beta.0",
-  "@docusaurus/preset-classic": "^2.0.0-beta.0",
+  "@docusaurus/core": "^2.0.0-beta.2",
+  "@docusaurus/preset-classic": "^2.0.0-beta.2",
```
Comment (slorber) [inline website/docs/installation.md:132]:
```suggestion
  "@docusaurus/core": "^2.0.0-beta.0",
  "@docusaurus/preset-classic": "^2.0.0-beta.0",
```

It's not really sustainable to update those docs after each release, that's why we added a comment:

> Please update to the latest Docusaurus 2 version shown at the top of the page, not what is shown below.

If you want it to display the actual beta to use, we should rather use MDX instead and read from the `package.json` file to be sure it stays in sync over time

**Review by slorber — CHANGES_REQUESTED** [review]
Thanks

Unfortunately we can't keep updating the docs on each release, we need a better solution to solve this (like using MDX)

File: website/docs/installation.md (hunk 2 above)
Comment (iamayushdas) [reply website/docs/installation.md:132]: Okay cool

**Pull request merged**